modules/cid_checker/main.tf

```diff
@@ -0,0 +1,72 @@
+data "aws_caller_identity" "current" {}
```

Comment:
I would ask to move away the data logic to a separated file.
This change does not affect logic but will stay with our common terraform structure logic.

Reply:
Created data.tf file.